docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Faces.adoc

```diff
@@ -0,0 +1,122 @@
+:module-url: https://raw.githubusercontent.com/wildfly/wildfly/main/jsf/multi-jsf-installer/src/main/resources
+
+[[Jakarta_Faces]]
+= Jakarta Faces Configuration
+
+[abstract]
```

Comment:
@jasondlee This tag results in the paragraph being rendered in html in italics, which is funky looking and out of sync with the rest of the Admin Guide. Can your remove this please?

Reply:
@bstansberry Done!